### DISCUSSION TITLE
Daily 05/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#31] [Daily]
Con respecto al proyecto:

¿Qué hice ayer?
¿Qué hice o haré hoy?
Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Ayer terminé una clase de Python, hoy tengo Javascript, Diseño, y también clases de App moviles con el ISPC. Por mi parte más tarde pienso seguir viendo lo de android y ver si puedo hacer algun project MVP al menos con un Crud.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve terminando de pulir la documentación y creando las issue de las historias de usuario.
Hoy agregue al IEEE las historias de usuario nuevas y, además cree la issue correspondientes a esas US. Ahora estoy viendo como realizar los diagramas de casos de uso y de clases para el proximo sprint. Por la tarde tengo clases de App Moviles del instituto.

**Comment by marceloedelgado** [comment]
Ayer estuve probando la conexion a la base de datos Firebase, levante el proyecto en google api y logre conectar la base de datos con un proyecto. Ahora estoy viendo como conectar la base de datos a un formulario de login. Tuve clases a la tarde de App Moviles del ISPC vimos la primer clase de Testing.
